sdk/securityinsights/Azure.ResourceManager.SecurityInsights/src/Generated/Models/EntityData.cs

```diff
@@ -7,14 +7,13 @@
 
 using Azure.Core;
 using Azure.ResourceManager.Models;
-using Azure.ResourceManager.SecurityInsights.Models;
 
-namespace Azure.ResourceManager.SecurityInsights
+namespace Azure.ResourceManager.SecurityInsights.Models
```

Comment:
Namespace change is also a breaking change. Please move this file to customized code and use `Azure.ResourceManager.SecurityInsights` namesapce while supressing the generated `Azure.ResourceManager.SecurityInsights.Models.EntityData`.

Reply:
fixed